hr_holidays_meeting_name/models/hr_holidays.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
It was not reflected on guidelines, but on template: https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3, so there was an agreement. You can check the discussion here: https://github.com/OCA/maintainer-tools/pull/91